content/en/docs/reference/command-line-tools-reference/feature-gates/DisableNodeKubeProxyVersion.md

```diff
@@ -10,8 +10,8 @@ stages:
     defaultValue: false
     fromVersion: "1.29"
     toVersion: "1.30"
-  - stage: beta
-    defaultValue: true
```

Comment:
Was this gate ever beta?

Reply:
According to https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md
It was beta in 1.31.0, then reverted in 1.31.1.

Reply:
Let's list those patch version - the implementation allows that level of detail.

Reply:
Not sure if this makes better sense.
Will try it anyway.